_sass/panel.scss

```diff
@@ -90,3 +90,32 @@ img.image {
 .panel-fill {
   width: 100%;
 }
+
+.panel.latest-content {
+  padding: 1rem 0;
+
+  .fixed-width {
+    display: flex;
+    justify-content: space-around;
```

Comment:
If the title of the blog post gets really long I think this will result in that taking up a lot more width than the event.
Is this actually the visual we want?
(My assumption is that we're aiming for them to each be centred in their own column and occupy half the width)
Maybe worth trying with one of the longer post-competition blog posts as the latest and seeing how it looks.

Reply:
Yeah with longer blog posts titles the width split was indeed not 50/50. f33ca38 addresses this